bitnami/thanos/values.yaml

```diff
@@ -1953,11 +1993,23 @@ ruler:
   ## K8s Security Context for Thanos Ruler pods
   ## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
   ##
-  securityContext:
+  podSecurityContext:
     enabled: true
     fsGroup: 1001
     runAsUser: 1001
 
+  ## K8s containers' Security Context for Thanos Ruler container
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+  ##
+  containerSecurityContext:
+    enabled: true
+    capabilities:
+      drop:
+      - ALL
```

Comment:
Could you clarify why this is needed?

Reply:
@miguelaeh I think it is redundant with `allowPriviliegeEscalation: true` if the container runtime always behaves the same (like docker and containerd)

Reply:
Removed it because of redundant definition. It depends highly on the cri if allowPrivilegeEscalation covers the drop all capabilities. The most common once do that.